library/src/main/java/org/xmtp/android/library/Conversations.kt

```diff
@@ -105,23 +105,23 @@ data class Conversations(
     }
 
     suspend fun newGroup(
-        accountAddresses: List<String>,
+        inboxIds: List<String>,
```

Comment:
This actually still takes addresses (and then we resolve them to `inbox_id`s internally). If you think it'd be useful we can also give you a version that takes `inbox_id`s.